js/index.js

```diff
@@ -39,4 +39,81 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+let circle = document.querySelector('#cta-img');
+circle.setAttribute('src', siteContent["cta"]["img-src"]);
+
+let widePic = document.querySelector('#middle-img');
+widePic.setAttribute('src', siteContent['main-content']["middle-img-src"]);
+
+let navBar = document.querySelectorAll('nav a');
+navBar[0].textContent = 'Services';
+navBar[1].textContent = 'Product';
+navBar[2].textContent = 'Vision';
+navBar[3].textContent = 'Features';
+navBar[4].textContent = 'About';
+navBar[5].textContent = 'Contact';
+
+navBar[0].style.color = 'green';
+navBar[1].style.color = 'green';
+navBar[2].style.color = 'green';
+navBar[3].style.color = 'green';
+navBar[4].style.color = 'green';
+navBar[5].style.color = 'green';
+
+let newNav1 = document.createElement('a');
+newNav1.textContent = 'yeet';
+newNav1.style.color = 'green';
+let newNav2 = document.createElement('a');
+newNav2.textContent = 'yeehaw';
+newNav2.style.color = 'green';
+
+let navParent = document.querySelector('nav');
+navParent.appendChild(newNav2);
+navParent.appendChild(newNav1);
+
+let mainHeader = document.querySelector('h1');
+mainHeader.textContent = "DOM Is Awesome";
+
+let topDisp = document.querySelector('.cta');
+topDisp.style.display = 'flex';
+topDisp.style.flexDirection = 'row';
+topDisp.style.justifyContent = 'center';
+topDisp.style.justifyContent = 'center';
+topDisp.style.alignContent = 'center';
+
+mainHeader.style.verticalAlign  = 'middle';
+mainHeader.style.textAlign  = 'center';
+mainHeader.style.display = 'flex';
+mainHeader.style.flexDirection = 'column';
+mainHeader.style.justifyContent = 'center';
+mainHeader.style.justifyContent = 'center';
+mainHeader.style.alignContent = 'center';
+
+
+
+
+
+let started = document.querySelector('button');
+started.textContent = "Get Started";
+
+let paraTitles = document.querySelectorAll('h4');
+paraTitles[0].textContent = 'Features';
+paraTitles[1].textContent = 'About';
+paraTitles[2].textContent = 'Services';
+paraTitles[3].textContent = 'Product';
+paraTitles[4].textContent = 'Vision';
+paraTitles[5].textContent = 'Contact';
+
+let paras = document.querySelectorAll('p');
+paras[0].textContent = 'Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.';
```

Comment:
Instead of copy/pasting the text, try:
tContentH[0].textContent = siteContent['main-content']['features-h4'];
tContentH[1].textContent = siteContent['main-content']['about-h4'];
tContentH[2].textContent = siteContent['main-content']['services-h4'];
tContentH[3].textContent = siteContent['main-content']['product-h4'];
tContentH[4].textContent = siteContent['main-content']['vision-h4'];

It will clean up your code, and make it more semantic/abstract.